R/log.R

```diff
@@ -82,10 +86,14 @@ log_config <- function(file = NA, log_name = NA, log_path = NA){
    set_log_element("lint_results", get_lint_results(file))
 }
 
-#' Cleans up log and does checks against elements
+#' Cleaning-up of log.rx object
```

Comment:
Can we also have something on why we think we should be cleaning up the environment?